tests/main/snap-confine/task.yaml

```diff
@@ -43,6 +43,10 @@ execute: |
 
     echo "Ensure apparmor profile for snap-confine is parsable"
     for f in /etc/apparmor.d/usr.lib.snapd.snap-confine*; do
+        # Breake the loop when the file does not exist
+        if [ ! -f "$f" ]; then
```

Comment:
I don't think this is supposed to happen on openSUSE. @zyga we don't ship s-c apparmor profiles on opensuse?

Reply:
Yes, that's correct. Unless something broke in packaging and suddenly we do.

Perhaps the loop should change to a find-style loop. Unfortunately shell shows the sharp edges all too often.